### DISCUSSION TITLE
1.7.10 Support possible?

### DISCUSSION THREAD

**Discussion by OMNIDROID2995** [mircokroon/minecraft-world-downloader#131] [Ideas]
We have a very complex server project that has been under construction for a few years. With the dimmensions this world has taken by now (over 10GB), the server sometimes doesn't manage to load certain things in time. Since there is a train track network hundreds of kilometers long and the trains can travel very fast, it is then sometimes too much with the cunk loading. The server is heavily modded. Is this downloader only for vanilla or would it not be a problem?

**Comment by mircokroon** [comment]
Well there isn't really a reason it could not be made to support 1.7.10, but it would take quite a lot of work for me to support it just for one server. I can't imagine many people still use versions this old.

**Reply by OMNIDROID2995** [reply]
Because we have already tried several times to move to a newer version, but also did not want to give up our years of work because a redesign of the map would catapult us into middle age (many of the 83 mods like to make problems). But no problem. I understand that it would be absolutely no easy undertaking. I will try to say goodbye to the bad game vServer providers with their low singlecore power. Maybe a dedicated rental server with Ryzen 5000 will do the trick. But thanks for your attention.

Translated with www.DeepL.com/Translator (free version)
